geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/sortedset/AbstractSortedSetRangeOptions.java

```diff
@@ -37,25 +39,32 @@
   }
 
   void parseLimitArguments(List<byte[]> commandElements, int commandIndex) {
-    if (!equalsIgnoreCaseBytes(commandElements.get(commandIndex - 2), bLIMIT)) {
+    String element = bytesToString(commandElements.get(commandIndex));
+    if (!equalsIgnoreCaseBytes(commandElements.get(commandIndex), bLIMIT)) {
+      System.out.println("DONAL: throwing because " + element + " is not LIMIT");
```

Comment:
remove the debug logging

Reply:
the debug logging is in a couple of places

Reply:
Thanks for spotting that. It's gone now.